flocker/testtools/skips.py

```diff
@@ -0,0 +1,45 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Utilities to help with skipping tests.
+"""
+
+def fail_if_skipped(test_item):
+    """
+    Check whether to fail the given test object if it will be skipped.
+
+    :param test_object: test method or class with a skip decorator.
+
+    :return: True if ``test_object`` is covered by environment variable
+        FLOCKER_TEST_NO_SKIPS, else False.
+    """
+    return False
+
+
+def skipUnless(condition, reason):
```

Comment:
These functions need suitable names. Ideally those names would read well, like how `skipUnless(SELENIUM_INSTALLED)` reads like "skip unless selenium is installed". One option is to have them called `skipIf` and `skipUnless`, like the unittest methods, and then only import lines must be changed. Another option is `skipOrFailUnless` and `skipOrFailIf`.